### PR TITLE
Internal compiler: first try find and use clang with version in postf…

### PR DESCRIPTION
**Pull request by proller**
…ix (clang-4.0)

### CODE REVIEW

**Comment by ekonkov** [comment]
autotest

File: debian/copy_clang_binaries.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ PATH="/usr/local/bin:/usr/local/sbin:/usr/bin:$PATH"
 LD=$(command -v gold || command -v ld.gold || command -v ld)
 
 if [ -z "$CLANG" ]; then
-    CLANG=$(which clang)
+    CLANG=$(which clang-4.0 || which clang)
```
Comment (alexey-milovidov) [inline debian/copy_clang_binaries.sh]: clang 5 (pre release candidate) and clang 6 (current trunk) are also Ok and should be preferred over clang 4.

**Comment by alexey-milovidov** [comment]
This breaks our current CI environment.

**Comment by alexey-milovidov** [comment]
Any clang version before 4.0 is strongly not recommended to use.

**Comment by proller** [comment]
autotest

**Pull request merged**